workflows/genetic-demux/index-star.nf

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+// parameters
+params.assembly  = 'Homo_sapiens.GRCh38.104'
+params.ref_dir   = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-104'
+params.ref_fasta = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-104/fasta/Homo_sapiens.GRCh38.dna.primary_assembly.fa.gz'
+params.ref_gtf   = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-104/annotation/Homo_sapiens.GRCh38.104.gtf.gz'
+
+STARCONTAINER = 'quay.io/biocontainers/star:2.7.9a--h9ee0642_0'
+
+process index_star{
+  container STARCONTAINER
+  publishDir "${params.ref_dir}/star_index", mode: 'copy'
+  memory "64.GB"
+  cpus "8"
+  input:
+    path ref_fasta
+    path ref_gtf
+  output:
+    path output_dir
+  script:
+    output_dir = "${params.assembly}"
```

Comment:
```suggestion
    output_dir = "${params.assembly}.star_idx"
```
I do feel like having some sort of identifier that it's a star index even if it's in that folder could be helpful for the output. Only because we have so many different indices with the same assembly and if you were to ever use the name of the folder down the line I think it would be more helpful rather than harmful to be more descriptive here.